Xamarin.PropertyEditing/Common/CommonValueConverter.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Xamarin.PropertyEditing.Common
```

Comment:
mmm, what's the point of this? (also, too many usings)

Reply:
It's a placeholder for a Known Type: https://github.com/xamarin/Xamarin.PropertyEditing/pull/345/files#diff-720d356387057b7d346b2638967f07deR30